util/process-doc/process-doc.go

```diff
@@ -44,6 +45,10 @@ func main() {
 	case *makeRust:
 		fmt.Print("// AUTOMATICALLY GENERATED BY `process-doc`.  DO NOT EDIT.\n")
 		fmt.Print("// Edit `util/formats/newfmt/types.go`, then in `util/process-doc` run `make install`.\n\n")
+
+	case *makeYaml:
+		fmt.Print("# AUTOMATICALLY GENERATED BY `process-doc -w -yaml`.  DO NOT EDIT.\n")
```

Comment:
It's a good idea to specify in this comment that the -yaml option was used, and it would be great if you could add the similar clarification to the two previous blocks while you're in here.

Probably you want to remove the -w in the text since there's no reason to assume one way or the other that it was used, and -w should not have any bearing on the output.